src/js/components/Header/VolumeControl.react.js

```diff
@@ -32,7 +32,7 @@ export default class VolumeControl extends Component {
     }
 
     getVolumeIcon(volume, muted) {
-        return muted || volume === 0 ? 'volume-off' : volume > 0.5 ? 'volume-up' : 'volume-down';
+        return muted || volume === 0 ? 'volume-off' : Math.pow(volume, 1 / 4) > 0.5 ? 'volume-up' : 'volume-down';
```

Comment:
Ah, yea. These values are based on the linear values. Good catch.

One more knitpick. You can take it or leave it: since we're now doing this twice, I think it'd be good to pull it out into a function (`LinearizeVolume` or something) to DRY it up. It's probably also a good idea to do that with the `Math.pow` (`ExponentiateVolume` or something), so it's clear why we're doing it.